### ISSUE TITLE
Autosave not working

### ISSUE THREAD

**Issue by Chaitanyabsprip** [olimorris/persisted.nvim#2]
I just installed the plugin and it seems like autosave is not working. If I am explicitly calling session save command and session load last command then it worked. but autosave not working

**Comment by olimorris** [comment]
The session is only written to disk when you leave Neovim and the `VimLeavePre` autocmd is triggered. If you haven't lazy loaded the plugin and `autosave = true` then the plugin creates the autocmd to save.

**Comment by Chaitanyabsprip** [comment]
I have lazy loaded the plugin

**Comment by Chaitanyabsprip** [comment]
So apparently it is auto-saving, just not auto-loading. Is that expected behavior?

**Comment by olimorris** [comment]
Yep that's right. You can use the `SessionLoad` command or could have an autocommand do it. It's a couple of lines of code to implement an auto-load so would be happy to do so if you wish to raise another issue.
